Assets/MixedRealityToolkit.Tests/TestUtilities/MixedReality.Toolkit.Tests.Utilities.nuspec

```diff
@@ -0,0 +1,33 @@
+﻿<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>Microsoft.MixedReality.Toolkit.Tests.Utilities</id>
+    <version>$version$</version>
+    <authors>Microsoft</authors>
+    <owners>Microsoft,MixedReality</owners>
+    <licenseUrl>https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/License.txt</licenseUrl>
+    <projectUrl>https://aka.ms/MRTK</projectUrl>
+    <iconUrl>https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Images/MRTK_Logo_NuGet.png</iconUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>Test Utilities for the Mixed Reality Toolkit.</description>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <releaseNotes>$releaseNotes$</releaseNotes>
+    <tags>Unity MixedReality MixedRealityToolkit MRTK</tags>
+    <dependencies>
+      <dependency id="Microsoft.MixedReality.Toolkit.Foundation" version="$version$" />
+    </dependencies>
+    <contentFiles>
+      <files include="any\any\.PkgSrc\**" buildAction="None" copyToOutput="false" />
+    </contentFiles>
+  </metadata>
+  <files>
+    <!-- There is nothing to include in the MRTK folder-->
```

Comment:
Can you clarify this comment a little more? Reading this, I think "oh, there's nothing to include, then why are we including mostly everything minus the meta/nuspec/props stuff?". If there's nothing, should this line even exist?

Reply:
Oh, I guess actually this would probably end up pulling in version.txt (and maybe some other things which are actually important)

Reply:
Yeah I kept it there just in case other files would get added.  I can clarify that in a new comment.